src/Sql/Sql/Replication/Cmdlet/NewAzureSqlDatabaseSecondary.cs

```diff
@@ -172,7 +172,7 @@ protected override IEnumerable<AzureReplicationLinkModel> GetEntity()
 
             // The database already exists
             throw new PSArgumentException(
-                string.Format(Resources.DatabaseNameExists, this.DatabaseName, this.PartnerServerName),
+                string.Format(Resources.DatabaseNameExists, GetEffectivePartnerDatabaseName(this.DatabaseName, this.PartnerDatabaseName), this.PartnerServerName),
```

Comment:
Thanks for fixing this! 
Now this will tell the user that "Database already exists" for the secondary (partner) database without mentioning that is failing because of the partner database. I wonder how much work it is to add a new error message (something like "PartnerDatabaseNameExists" instead of using the existing DatabaseNameExists